packages/kbn-ui-framework/src/components/vertical_rhythm/_index.scss

```diff
@@ -1,6 +1,6 @@
-$verticalRhythmSpacing: 10px;
-$verticalRhythmSmallSpacing: 5px;
-$verticalRhythmLargeSpacing: 20px;
-$verticalRhythmXLargeSpacing: 40px;
+$verticalRhythmSpacing: $euiSize;
```

Comment:
Only thing that really scares me is who knows what absolute pixel heights were constructed assuming so sort of constant from these.

I'm not worried about the visual difference though. I based it the off intended usage I saw and mapped against our equivalent. These were used similar to our spacer components and not for pixel precision.

KUI's base was 10 and that one was the most common vertical spacing selector in Kibana. The equiv on our side is EuiSpacer size="m". It's most often used to provide distance between larger blocks and panels. Here's a good example where at least spacing in some mixed views is getting at least some semblance of consistency.

![image](https://user-images.githubusercontent.com/324519/49921218-ba8ebe80-fe61-11e8-89d0-36a13c008ead.png)

`.kuiVerticalRhythmSmall` is mostly used for separation of form elements and checkboxes from what I could see. In all the cases I checked the spacing made sense and was closer to our usage in EUI. Below are some screens showing common usage of the smaller sizes.

![image](https://user-images.githubusercontent.com/324519/49920860-92eb2680-fe60-11e8-85d0-4c2e9dd0d41e.png)

![image](https://user-images.githubusercontent.com/324519/49920864-97afda80-fe60-11e8-81cb-7139e002ccd3.png)

This is all pretty quick damage. Spacing is one of the quick things I think we can change in KUI to make things _feel_ a little closer. For a half minute I thought... hey, why don't I just extend EUI forms into KUI ones, but I backed off because I kind of want to be able to look at the page and still know what needs to be removed. Ultimately, I think we're in the single digit months of how long KUI will actually be around.

This stuff ultimately I think is a band aid.

Reply:
Got it! It's one of those situations where I don't know if it's intentional or not. Looks like it was thoroughly thought through. 🙇‍♀️